### PR TITLE
Add commissioning desks to snippets

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Much simpler than I expected thanks to the tag field 😃  It's coming through quite nicely:

screen shot 2017-10-10 at 13 45 45

NB: I've added the ability to filter by type in the tag search, it is not returning any result in preview but I tested with CAPI prod and works fine!

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
Mandatory screenshot:

screen shot 2017-10-10 at 14 02 21

File: public/js/components/AtomEdit/CustomEditors/GuideEditor.js [hunk 1]
```diff
@@ -28,6 +29,11 @@ export class GuideEditor extends React.Component {
               <GuideItem onFormErrorsUpdate={this.props.onFormErrorsUpdate} />
             </FormFieldArrayWrapper>
           </ManagedField>
+          <ManagedField fieldLocation="commissioningDesks" name="Commissioning desks" updateData={this.onDesk}>
```
Comment (tomrf1) [inline public/js/components/AtomEdit/CustomEditors/GuideEditor.js]: Where is `onDesk` coming from?

Comment (regiskuckaertz) [reply public/js/components/AtomEdit/CustomEditors/GuideEditor.js]: ha! good catch, I removed it but forgot to remove the attribute too.

**Comment by tomrf1** [comment]
👍 looks good, though I wonder if this field should be in the same section as the title?

**Comment by regiskuckaertz** [comment]
Oh, I did not think about this one. So it would be common across all atoms, right?

**Comment by tomrf1** [comment]
The field is at the top level of the atom model, though perhaps it will only be used for snippets. Might be worth asking Chris M/W if they have an opinion.

**Comment by regiskuckaertz** [comment]
Mariot spoke and so will I move that as you recommended 🌟

**Comment by regiskuckaertz** [comment]
There it s:

screen shot 2017-10-10 at 16 39 22

**Comment by tomrf1** [comment]
Perfect! 👍 

picture 8

**Pull request merged**